salt/modules/chocolatey.py

```diff
@@ -36,9 +36,13 @@ def __virtual__():
     salt-minion running as SYSTEM.
     '''
     if not salt.utils.is_windows():
-        return (False, 'Cannot load module chocolatey: Chocolatey requires Windows')
-    elif __grains__['osrelease'] in ('XP', '2003Server'):
-        return (False, 'Cannot load module chocolatey: Chocolatey requires Windows Vista or later')
+        return (False, 'Cannot load module chocolatey: Chocolatey requires '
+                       'Windows')
+
+    if __grains__['osrelease'] in ('XP', '2003Server'):
+        return (False, 'Cannot load module chocolatey: Chocolatey requires '
+                       'Windows Vista or later')
```

Comment:
@twangboy it still works in 2003 (for how much longer though is a good question).